parcel_test.go

```diff
@@ -30,58 +27,79 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
-	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	testParcel := getTestParcel()
+	id, err := store.Add(testParcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
+
+	tmpParcel, err := store.Get(id)
+	require.NoError(t, err)
+
+	assert.Equal(t, testParcel.Client, tmpParcel.Client)
+	assert.Equal(t, testParcel.Status, tmpParcel.Status)
+	assert.Equal(t, testParcel.Address, tmpParcel.Address)
+	assert.Equal(t, testParcel.CreatedAt, tmpParcel.CreatedAt)
 
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	err = store.Delete(id)
+	require.NoError(t, err)
 
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(id)
+	require.Equal(t, err, sql.ErrNoRows)
```

Comment:
Лучше использовать функции для работы с ошибками
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```

Reply:
оу, не знал о такой
спасибо, исправил